streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyContextImpl.java

```diff
@@ -75,8 +75,6 @@ public void close() {}
         }
     };
 
-    private long streamTime = RecordQueue.UNKNOWN;
```

Comment:
Can you fix `RecordQueue.UNKNOWN` as a side thing, too?

Reply:
You mean, set it to `ConsumerRecord.NO_TIMESTAMP`, right?

Reply:
Yes.